### PR TITLE
Clean directories for upgrade checks

### PR DESCRIPTION
**Pull request by dimberman**
As we plan to add many more upgrade checks soon,
this PR allows for compartmentalization of checks

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).

### CODE REVIEW

**Review by ashb — COMMENTED** [review]
Change conn_id/ to db/ or something like that -for other checks that might look at contents of db (I don't see if having anything else under conn_id/)?

**Review by kaxil — APPROVED** [review]

File: tests/upgrade/rules/test_base_rule.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # under the License.
 
 from airflow.upgrade.rules import get_rules
-from airflow.upgrade.rules.conn_type_is_not_nullable import ConnTypeIsNotNullableRule
+from airflow.upgrade.rules.conn_id.conn_type_is_not_nullable import ConnTypeIsNotNullableRule
```
Comment (ashb) [inline tests/upgrade/rules/test_base_rule.py:19]:
```suggestion
from airflow.upgrade.rules.db.conn_type_is_not_nullable import ConnTypeIsNotNullableRule
```

**Review by ashb — COMMENTED** [review]
This needs work -- points of the code are still referring to the old paths (`from airflow.upgrade.rules.renamed_classes import ALL` for example) -- if this passes tests then our tests are also not good enough :)

**Comment by turbaszek** [comment]
@dimberman please target `v1-10-stable`

**Pull request closed without merging**